SmartDeviceLink/SDLFileManager.m

```diff
@@ -40,7 +40,7 @@ @interface SDLFileManager ()
 @property (weak, nonatomic) id<SDLConnectionManagerType> connectionManager;
 
 // Remote state
-@property (copy, nonatomic, readwrite) NSMutableSet<SDLFileName *> *mutableRemoteFileNames;
+@property (strong, nonatomic, readwrite) NSMutableSet<SDLFileName *> *mutableRemoteFileNames;
```

Comment:
Yes, these are private properties, so we can modify them if desired. Warnings appear in Xcode 8.3b2 if they are left as is.